src/test/ui/proc-macro/dollar-crate-issue-62325.stdout

```diff
@@ -59,54 +59,54 @@ PRINT-ATTR RE-COLLECTED (DISPLAY): struct B (identity ! ($crate :: S)) ;
 PRINT-ATTR INPUT (DEBUG): TokenStream [
     Ident {
         ident: "struct",
-        span: #8 bytes(LO..HI),
+        span: #10 bytes(LO..HI),
```

Comment:
Why did those change?

Reply:
We pre-create more `SyntaxContext`s in `proc_macro_server.rs`, so their indices may grow.

Reply:
Missed https://github.com/rust-lang/rust/pull/64690/files#diff-e8273b2e003e0ebce4ea778a840258ddR368